README.md

```diff
@@ -1,36 +1,74 @@
-# Kiwix for iOS & macOS
+# Kiwix for Apple iOS & macOS
 
-This is the home for Kiwix apps on iOS and macOS.
+This is the home for Kiwix apps for Apple iOS and macOS.
 
 [![CodeFactor](https://www.codefactor.io/repository/github/kiwix/apple/badge)](https://www.codefactor.io/repository/github/kiwix/apple)
 [![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
 <img src="https://img.shields.io/badge/Swift-5.2-orange.svg" alt="Drawing="/>
 
-### Mobile app for iPads & iPhones ###
+## Download
+
+Kiwix apps are made available primarily via the [Mac App
+Store](https://macos.kiwix.org).
+
+Most recent versions of Kiwix support the three latest versions of the
+OSes (either iOS or macOS). Older versions of Kiwix being still
+downloadable for older versions of macOS and iOS on the Mac App Store.
+
+### iPads & iPhones ###
 - Download the iOS mobile app on the [App Store](https://ios.kiwix.org)
 
-### Kiwix Desktop for macOS ###
+### macOS ###
 - Download Kiwix Desktop on the [Mac App Store](https://macos.kiwix.org)
 - Download Kiwix Desktop [DMG file](https://download.kiwix.org/release/kiwix-desktop-macos/kiwix-desktop-macos.dmg)
 
-## Developers
+## Develop
+
+Kiwix developers use to work with cutting-edge versions of both macOS
+and Xcode. [Continuous
+integration](https://en.wikipedia.org/wiki/Continuous_integration)
+secures that the whole project still compiles on the next to last
+version of macOS with latest version of Xcode distributed on it.
+
+### CPU Architectures
+
+Kiwix compiles on both macOS with x86_64 or arm64 (M1, M2, ... family).
+
+Kiwix for iOS and macOS can run, in both cases, on x86_64 or arm64.
```

Comment:
This makes little sense in regard to the previous sentence. Is there a different between _compiles_ and _runs_? Why aren't the iOS archs listed? iOS actually runs x86_64 only within the simulator (and there's no iOS x86_64 device)

Reply:
I want cross-compilation targets to be explicit to avoid any ambiguity.